packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -817,12 +834,18 @@ export class SDKClient {
    * @param callerName
    * @param interactingEntity
    */
-  public deleteFile = async (fileId: FileId, requestId?: string, callerName?: string, interactingEntity?: string) => {
+  async deleteFile(fileId: FileId, requestId: string, callerName: string, interactingEntity: string): Promise<void> {
     // format request ID msg
     const currentDateNow = Date.now();
     const requestIdPrefix = formatRequestIdMessage(requestId);
 
     try {
+      // TODO: Limiting was missing here, not sure if it was intentional
+      const shouldLimit = this.hbarLimiter.shouldLimit(currentDateNow, SDKClient.transactionMode, callerName);
```

Comment:
Was intentional.
We always want to clean up fies so we don't want the rate limiter to block us in this case.